shardingsphere-infra/shardingsphere-infra-optimize/src/main/java/org/apache/shardingsphere/infra/optimize/context/OptimizeContextFactory.java

```diff
@@ -156,7 +156,7 @@ private void initProperties(final DatabaseType databaseType) {
     }
     
     private RelOptCluster newCluster() {
-        RelOptPlanner planner = new VolcanoPlanner();
+        RelOptPlanner planner = new VolcanoPlanner(null, null);
```

Comment:
@wgy8283335 Why add null param?

Reply:
At first, I thought they are different.
After I've checked these methods, they are the same. 
I will fix this.